framework/include/media/MediaRecorder.h

```diff
@@ -160,9 +160,10 @@ class MediaRecorder
 	 * @details @b #include <media/MediaRecorder.h>
 	 * This function is sync call apis
 	 * It sets the user's function
+	 * @return The result of setting the observer
 	 * @since TizenRT v2.0 PRE
 	 */
-	void setObserver(std::shared_ptr<MediaRecorderObserverInterface> observer);
+	recorder_result_t setObserver(std::shared_ptr<MediaRecorderObserverInterface> observer);
```

Comment:
FYI) I fixed setObserver implementation. Please refer this PR : https://github.com/Samsung/TizenRT/pull/1675